gadopt/energy_solver.py

```diff
@@ -1,34 +1,28 @@
-r"""This module provides a fine-tuned solver class for the energy conservation equation.
-Users instantiate the `EnergySolver` class by providing relevant parameters and call
-the `solve` method to request a solver update.
+r"""This module provides a minimal solver for generic transport equations, which may
```

Comment:
In light of the change here to energy solver inheriting from a generic setup, should the file name be changed from energy_solver to something different?

Reply:
I have been considering changing file names, but I stopped myself from doing so. If developers are for it, then why not `transport_solvers.py`?